backend/src/schema/resolvers/follow.spec.js

```diff
@@ -41,44 +70,57 @@ describe('follow', () => {
   describe('follow user', () => {
     describe('unauthenticated follow', () => {
       it('throws authorization error', async () => {
-        const client = new GraphQLClient(host)
-        await expect(client.request(mutationFollowUser('u2'))).rejects.toThrow('Not Authorised')
+        authenticatedUser = null
+        const { errors } = await mutate({
+          mutation: mutationFollowUser('u2'),
+        })
+        expect(errors[0]).toHaveProperty('message', 'Not Authorised!')
       })
     })
 
     it('I can follow another user', async () => {
-      const res = await clientUser1.request(mutationFollowUser('u2'))
-      const expected = {
-        follow: true,
-      }
-      expect(res).toMatchObject(expected)
+      const { data: result } = await mutate({
+        mutation: mutationFollowUser(user2.id),
+      })
+
+      const expected = { follow: true }
+      expect(result).toMatchObject(expected)
+
+      const {
+        data: { User },
+      } = await query({
+        query: `{
+          User(id: "${user2.id}") {
+            followedBy { id }
+            followedByCurrentUser
+          }
+        }`,
+      })
 
-      const { User } = await clientUser1.request(`{
-        User(id: "u2") {
-          followedBy { id }
-          followedByCurrentUser
-        }
-      }`)
       const expected2 = {
-        followedBy: [{ id: 'u1' }],
+        followedBy: [{ id: user1.id }],
         followedByCurrentUser: true,
       }
       expect(User[0]).toMatchObject(expected2)
     })
 
     it('I can`t follow myself', async () => {
-      const res = await clientUser1.request(mutationFollowUser('u1'))
-      const expected = {
-        follow: false,
-      }
-      expect(res).toMatchObject(expected)
+      const { data: result } = await mutate({
+        mutation: mutationFollowUser(user1.id),
+      })
+      const expected = { follow: false }
```

Comment:
```suggestion
      const expected = { follow: null }
```

I think it's good practice to respond with `null` when nothing has been mutated in any way (e.g. nothing has been deleted).

Reply:
@roschaefer probably idea is good however this change requires schema update because of the mutation result is defined as `Boolean!` value. As we're about to change the schema significantly with further steps this change doesn't bring a lot of value IMO at the moment.